src/test/StkAaveRetrievalE2E.t.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// testing libraries
+import "@forge-std/Test.sol";
+
+// contract dependencies
+import "../external/aave/IStaticATokenLM.sol";
+import "../external/aave/IAaveIncentivesController.sol";
+import {GovHelpers} from "@aave-helpers/GovHelpers.sol";
+import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
+import {StkAaveRetrieval} from "../StkAaveRetrieval.sol";
+import {ProposalPayload} from "../ProposalPayload.sol";
+import {DeployMainnetProposal} from "../../script/DeployMainnetProposal.s.sol";
+import "@openzeppelin/token/ERC20/IERC20.sol";
+
+contract StkAaveRetrievalE2ETest is Test {
+    address public constant AAVE_WHALE = 0xBE0eB53F46cd790Cd13851d5EFf43D12404d33E8;
+
+    uint256 public proposalId;
+
+    StkAaveRetrieval private stkAaveRetrieval;
+    ProposalPayload private proposalPayload;
```

Comment:
You can remove this line too. ProposalPayload is already initialized and only used in setup()